python/kserve/docs/V1beta1ExplainerSpec.md

```diff
@@ -30,13 +30,13 @@ Name | Type | Description | Notes
 **min_replicas** | **int** | Minimum number of replicas, defaults to 1 but can be set to 0 to enable scale-to-zero. | [optional] 
 **node_name** | **str** | NodeName is a request to schedule this pod onto a specific node. If it is non-empty, the scheduler simply schedules this pod onto that node, assuming that it fits resource requirements. | [optional] 
 **node_selector** | **dict(str, str)** | NodeSelector is a selector which must be true for the pod to fit on a node. Selector which must match a node&#39;s labels for the pod to be scheduled on that node. More info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/ | [optional] 
-**overhead** | [**dict(str, ResourceQuantity)**](ResourceQuantity.md) | Overhead represents the resource overhead associated with running a pod for a given RuntimeClass. This field will be autopopulated at admission time by the RuntimeClass admission controller. If the RuntimeClass admission controller is enabled, overhead must not be set in Pod create requests. The RuntimeClass admission controller will reject Pod create requests which have the overhead already set. If RuntimeClass is configured and selected in the PodSpec, Overhead will be set to the value defined in the corresponding RuntimeClass, otherwise it will remain unset and treated as zero. More info: https://git.k8s.io/enhancements/keps/sig-node/20190226-pod-overhead.md This field is alpha-level as of Kubernetes v1.16, and is only honored by servers that enable the PodOverhead feature. | [optional] 
+**overhead** | [**dict(str, ResourceQuantity)**](ResourceQuantity.md) | Overhead represents the resource overhead associated with running a pod for a given RuntimeClass. This field will be autopopulated at admission time by the RuntimeClass admission controller. If the RuntimeClass admission controller is enabled, overhead must not be set in Pod create requests. The RuntimeClass admission controller will reject Pod create requests which have the overhead already set. If RuntimeClass is configured and selected in the PodSpec, Overhead will be set to the value defined in the corresponding RuntimeClass, otherwise it will remain unset and treated as zero. More info: https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/688-pod-overhead/README.md This field is alpha-level as of Kubernetes v1.16, and is only honored by servers that enable the PodOverhead feature. | [optional] 
```

Comment:
These are generated from the [PodSpec](https://github.com/kserve/kserve/blob/master/pkg/apis/serving/v1beta1/podspec.go#L232), you did regenerate the doc from swagger.json right?

Reply:
I was only trying to fix as many of the links as I could find. I am new to KServe so I am not yet familiar with the codegen process.

The intention of this PR is to provide a tool to identify broken links. I don't have the knowledge to fix them all :-)

Reply:
Thanks for the great tool!! Actually I just bumped up the kube version a few days ago and that seems like fixed links here so we could just run the [codegen](https://github.com/kserve/kserve/tree/master/hack/python-sdk#generate-python-sdk) to update the docs.

Reply:
Okay. I just ran the codegen locally and the links are indeed fixed. But a few new ones don't work :-\

```YAML
Checked 1253 links (530 unique URLs) in 187 Markdown files.

README.md:3: https://coveralls.io/github/kserve/kserve?branch=master -> 404
docs/samples/README.md:36: https://github.com/kserve/kserve/tree/master/python/lightgbm -> 404
docs/samples/explanation/alibi/moviesentiment/README.md:371: /docs/samples/explanation/alibi/moviesentiment/./kfserving_text_explainer.ipynb -> 404
python/kserve/docs/V1alpha1ClusterServingRuntime.md:11: /python/kserve/docs/.md -> 404
python/kserve/docs/V1alpha1ModelSpec.md:8: /python/kserve/docs/ResourceQuantity.md -> 404
python/kserve/docs/V1alpha1ServingRuntime.md:11: /python/kserve/docs/.md -> 404
python/kserve/docs/V1alpha1TrainedModel.md:11: /python/kserve/docs/V1alpha1TrainedModelStatus.md -> 404
python/kserve/docs/V1beta1CustomExplainer.md:24: /python/kserve/docs/V1PodOS.md -> 404
python/kserve/docs/V1beta1CustomExplainer.md:25: /python/kserve/docs/ResourceQuantity.md -> 404
python/kserve/docs/V1beta1CustomPredictor.md:24: /python/kserve/docs/V1PodOS.md -> 404
python/kserve/docs/V1beta1CustomPredictor.md:25: /python/kserve/docs/ResourceQuantity.md -> 404
python/kserve/docs/V1beta1CustomTransformer.md:24: /python/kserve/docs/V1PodOS.md -> 404
python/kserve/docs/V1beta1CustomTransformer.md:25: /python/kserve/docs/ResourceQuantity.md -> 404
python/kserve/docs/V1beta1ExplainerSpec.md:33: /python/kserve/docs/V1PodOS.md -> 404
python/kserve/docs/V1beta1ExplainerSpec.md:34: /python/kserve/docs/ResourceQuantity.md -> 404
python/kserve/docs/V1beta1PodSpec.md:24: /python/kserve/docs/V1PodOS.md -> 404
python/kserve/docs/V1beta1PodSpec.md:25: /python/kserve/docs/ResourceQuantity.md -> 404
python/kserve/docs/V1beta1PredictorSpec.md:33: /python/kserve/docs/V1PodOS.md -> 404
python/kserve/docs/V1beta1PredictorSpec.md:34: /python/kserve/docs/ResourceQuantity.md -> 404
python/kserve/docs/V1beta1TransformerSpec.md:30: /python/kserve/docs/V1PodOS.md -> 404
python/kserve/docs/V1beta1TransformerSpec.md:31: /python/kserve/docs/ResourceQuantity.md -> 404

ERROR: Found 21 invalid Markdown links

```